renderer/components/action-bar/record-button.js

```diff
@@ -42,7 +42,7 @@ class RecordButton extends React.Component {
           }
         };
       }, err => {
-        console.error('The following error occured: ' + err.name);
+        console.error('An error occurred when trying to get audio levels: ', err);
```

Comment:
You don't need the space here as `console` adds space between arguments.

`levels: '` => `levels:'`

Reply:
Wouldn't it be nice with a linter rule for this? @sindresorhus

Reply:
@albinekb Agreed! Can you open an issue https://github.com/sindresorhus/eslint-plugin-unicorn ?

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/issues/175